src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/repository/EventManagerAuditRepository.kt

```diff
@@ -6,4 +6,6 @@ import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.EventManagerEntity
 
 interface EventManagerAuditRepository : CrudRepository<EventManagerAuditEntity, Long> {
   fun findByEventManager(eventManager: EventManagerEntity): List<EventManagerAuditEntity>
+
+  fun findFirstByEventManagerOrderByCreatedDateDesc(eventManager: EventManagerEntity): EventManagerAuditEntity?
```

Comment:
It's a shame that we don't have some link between the eventmanager table entries and eventmanageraudit table entries - using the created date seems like a hack.

Reply:
they are linked by a column on the event manager audit table, https://github.com/ministryofjustice/hmpps-workload/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/EventManagerAuditEntity.kt#L40. The ordering is to get the latest audit record

Reply:
Ah yeah.. as it's an audit table it's possibly got >1 entry per event manager. Ok good.